lib/common/constant.go

```diff
@@ -18,6 +18,9 @@ const (
 	// GenesisBlockHeight set the block height of genesis block
 	GenesisBlockHeight uint64 = 1
 
+	// FirstConsensusBlockHeight is used for calculating block time
+	FirstConsensusBlockHeight uint64 = 2
```

Comment:
Under our current implementation, `FirstConsensusBlockHeight` must be `GenesisBlockHeight + 1`, right? We don't need to define new one, I think.

Reply:
When you see this code, is this understandable than `FirstConsensusBlockHeight`?
```
func calculateAverageBlockTime(firstConsensusBlockTime time.Time, blockHeight uint64) time.Duration {
	height := blockHeight - (common.GenesisBlockHeight + 1)
	sinceGenesis := time.Now().Sub(firstConsensusBlockTime)

	if height == 0 {
		return sinceGenesis
	} else {
		return sinceGenesis / time.Duration(height)
	}
}

```

Reply:
@Charleslee522 If `FirstConsensusBlockHeight` can not be `GenesisBlockHeight + 1`, it will be :)